packages/shopify-codemod/test/transforms/implicit-coercion-to-explicit.test.js

```diff
@@ -0,0 +1,8 @@
+import 'test-helper';
+import transform from 'implicit-coercion-to-explicit';
```

Comment:
Is it better to give the imported transform a more explicit name? Most of the other transforms currently use a camel cased version of the transform file.